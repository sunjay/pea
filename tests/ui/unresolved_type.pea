fn foo(x: does_not_exist) {}
