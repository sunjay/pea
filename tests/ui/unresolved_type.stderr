[tests/ui/unresolved_type.pea:1:11-1:24] error: cannot find name `does_not_exist` in this scope

error: aborting due to 1 previous error

